src/style.css

```diff
@@ -1,3 +1,66 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font in the `font-family` property for better compatibility across different browsers and systems.